x/docs/track.rb

```diff
@@ -1,41 +0,0 @@
-module X
-  module Docs
-    # Enhances the track-specific documentation, adding
-    # defaults where missing and embellishments that encourage
-    # people to submit improvements.
-    class Track
```

Comment:
Now that we use the Trackler gem to grab track-specific content, we don't use this class. Instead, we have a [`Track` presenter](https://github.com/exercism/exercism.io/blob/master/app/presenters/track.rb) that essentially wraps the Trackler track.